.gitignore

```diff
@@ -79,10 +79,11 @@ reports/
 *.mypy*
 
 # VS Code
-java/.vscode
+.vscode/
```

Comment:
I just remembered that we have https://github.com/uyuni-project/uyuni/blob/master/.vscode/settings.json. That's something @uyuni-project/frontend added, so I guess we need to exclude it?

Reply:
The frontend part of the codebase intentionally includes that file in the repo, it's necessary for Typescript support etc to work out of the box since our setup is slightly unorthodox due to historical reasons. It would be preferable if we could keep the file as-is. What's the specific conflict in this case, perhaps there's another way around the problem?

Reply:
Talking about specific settings, auto formatting enabled has bitten me a few times when I used VSCode. Nothing I couldn't turn off, but maybe we should discuss IDE settings that apply to all languages.

Reply:
I think that's a fair point, if there's cases where that is unwanted we should turn that off. Are there any other settings that cause you problems? Would [limiting formatting to specific languages](https://stackoverflow.com/questions/44831313/how-to-exclude-file-extensions-and-languages-from-format-on-save-in-vscode) work for you?